### DISCUSSION TITLE
Is it possible to not use the global css values on body tag ?

### DISCUSSION THREAD

**Discussion by HassanZahirnia** [tusen-ai/naive-ui#2953] [Q&A] [answered]
Hey, I was just using Naive UI alongside Tailwind CSS and I noticed some weird behavior regarding the font size and style. I was scratching my head for sometime and just realized Naive UI applies certain css values to `body` which causes conflict with Tailwind:
```html
<style cssr-id="n-global">
body {

 margin: 0;
 font-size: 14px;
 font-family: v-sans, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
 line-height: 1.6;
 -webkit-text-size-adjust: 100%;
 -webkit-tap-highlight-color: transparent;

}

body input {

 font-family: inherit;
 font-size: inherit;
 
}</style>
```

I wonder if it's possible to have Naive UI not apply these values ?
In my case I want Tailwind CSS to be in charge of the body styles like `font-size`, `font-family`, `line-height`. 

Note that I'm aware of this [snippet](https://www.naiveui.com/en-US/light/docs/style-position) and I'm already applying it:
```js
const meta = document.createElement('meta')
meta.name = 'naive-ui-style'
document.head.appendChild(meta)

vueApp.mount('#app')
```

**Comment by 07akioni** [comment, marked as answer]
I think what you said is right.

I've make it optional in 55b13328f508474350906e5bfbafee9b211a0f26

It'll be released in next version.

**Reply by HassanZahirnia** [reply]
Wow that was a quick and delightful response ! Thanks for making it happen :)
What is usually the ETA on the releases ?

**Reply by 07akioni** [reply]
Possibly in next few days (less than 7).

**Comment by NisuSan** [comment]
@07akioni, hi, maybe you know how to apply this code for nuxt 3

```javascript
const meta = document.createElement('meta')
meta.name = 'naive-ui-style'
document.head.appendChild(meta)

vueApp.mount('#app')
```
